lib/Db/StatisticsDao.php

```diff
@@ -144,6 +144,53 @@ public function getNumberOfMessagesGrouped(array $mailboxes, array $emails): arr
 		return $data;
 	}
 
+	public function getSubjects(array $mailboxes, array $emails): array {
```

Comment:
nit: annotate the array shape for better static analysis

Reply:
The method is unused and gone.